### PR TITLE
Circular imports

### PR DESCRIPTION
**Pull request by gnhenderson**
- Circular imports were preventing the app from being configured correctly so moved app creation to the package `__init__.py`.
- All modules now import the `app` from `slingshotapi/__init__.py` *except* the `helpers/managedb.py` module. If it doesn't import `app` from the `server.py` module routes fail. I don't know why. This is going to need fixed later on.

### CODE REVIEW

**Comment by gjtempleton** [comment]
LGTM

File: slingshotapi/helpers/managedb.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 from flask_script import Manager
 from flask_migrate import Migrate, MigrateCommand
 
-from slingshotapi.models import app, db
+from slingshotapi.server import app
```
Comment (balintzs) [inline slingshotapi/helpers/managedb.py]: Import `app` from the `slingshotapi` package to keep it consistent everywhere.

File: slingshotapi/service.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+from slingshotapi.models import db, Project, Service, Deployment, Version
```
Comment (gnhenderson) [inline slingshotapi/service.py]: true - that's for something else from another branch. It doesn't need to be in this PR.

File: slingshotapi/helpers/managedb.py (hunk 1 above)
Comment (gnhenderson) [inline slingshotapi/helpers/managedb.py]: discussed with Balint offline

**Comment by balintzs** [comment]
LGTM

**Pull request merged**